### PR TITLE
Us33953 new

### PR DESCRIPTION
**Pull request by mandeep-singh-lab49**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: deployment/service/ContractService.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class ContractService {
       id: contractId,
       address: contractAddress,
       timestamp: new Date().toISOString(),
+      hash: "",
```
Comment (amandeeprehal-lab49) [inline deployment/service/ContractService.ts:90]: why this change Mandeep?

File: e2e-test.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    exit 0;
+fi
+
+echo "********************  Deployment *************************************"
+
+echo "Starting deployment for CONTRACT_NAME  " $CONTRACT_NAME;
+echo "Starting deployment for CONTRACT_TYPE  " $CONTRACT_TYPE;
+
+echo "Running contract deployment ........ " $CONTRACT_NAME;
+echo CONTRACT_NAME=Pair >> .env;
```
Comment (amandeeprehal-lab49) [inline e2e-test.sh:35]: Hardcoding for now?

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Review by shivamgarglab49 — APPROVED** [review]

**Comment by mandeep-singh-lab49** [comment]
creating another pr to test the report url issue and closing it without merge
